lua/entities/gmod_wire_expression2/core/custom/cl_cfc_e2_lib.lua

```diff
@@ -43,4 +43,80 @@ E2Helper.Descriptions["indexOf(r:a)"]          = "Returns the first index at whi
 E2Helper.Descriptions["indexOf(r:v)"]          = "Returns the first index at which the value is stored in the array, and 0 if it was not found."
 
 -- Vehicle Functions
-E2Helper.Descriptions["ejectPodTo(e:v)"]       = "Ejects driver of seat to desired location."
\ No newline at end of file
+E2Helper.Descriptions["ejectPodTo(e:v)"]       = "Ejects driver of seat to desired location."
+
+-- Vector 2 Functions
```

Comment:
Are you using spaces for indentation here?

Reply:
Yes